Repository/AbstractPhpcrRepository.php

```diff
@@ -147,6 +148,15 @@ public function add($path, $resource)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function reorder($sourcePath, $position)
+    {
+        Assert::greaterThanEq($position, 0, 'Reorder position cannot be negative, got: %s');
+        $this->reorderNode($sourcePath, $position);
```

Comment:
i guess we should. but the abstract phpcr repository can theoretically
still be extended by others... but if they do, i don't care if we break
their code - they hopefully know what they are doing and test and notice
the problem then.